contracts/managers/allocators/BinaryAllocator.sol

```diff
@@ -497,4 +507,21 @@ contract BinaryAllocator is
             )
         );
     }
+
+    /*
+     * Check that passed component array contains one component, else revert.
+     *
+     * @param  _array     Array to be evaluated
+     */
+    function componentArrayContainsOneComponent(
```

Comment:
Naming suggestion: `validateSingleItemArray`. Would also consider putting this into `AddressArrayUtils`